loco.py

```diff
@@ -1,66 +1,98 @@
+#!/usr/bin/env python3
+
+import re
 import heapq
 import sys
 
 
-class LogFile(object):
+class Pushable:
```

Comment:
I think we could basically use a deque (https://docs.python.org/3.3/library/collections.html#collections.deque) instead of this which also takes care of several continuation lines